src/super_gradients/__init__.py

```diff
@@ -1,3 +1,5 @@
+__version__ = "3.2.0"
+
```

Comment:
Version needs to be at the beginning because of circular loops.
This would not be required if we don't automatically check the deprecated versions in runtime (see other comments), but I also don't think it should be our criteria to chose or not to test on runtime.

Reply:
I experienced this as well!